src/modelApiMethods.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable consistent-return */
+export const submitcomment = async (url, appID, character) => {
+  const name = document.getElementById('name').value;
+  const comment = document.getElementById('textComment').value;
+  /* eslint-disable no-underscore-dangle */
+  if (name !== '' && comment !== '') {
+    const response = await fetch(`${url}${appID}/comments/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        item_id: `${character._id}`,
+        username: `${name}`,
+        comment: `${comment}`,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    /* eslint-disable no-use-before-define */
```

Comment:
`Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice. Instead of disabling `eslint`, it will be nice to fix the issue since it a requirement for this project.